Task07.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+enum Genre {
+    Comedy,
+    Action,
+    Horror
+};
+
+struct Movie {
+    char name[100];

```

Comment:
It would be better to not use numbers. Instead use global constant like const int MAX_NAME_SIZE=100